src/main/java/de/blau/android/bookmarks/BookmarkIO.java

```diff
@@ -0,0 +1,59 @@
+package de.blau.android.bookmarks;
+
+import android.content.Context;
+
+import java.util.ArrayList;
+
+import de.blau.android.osm.ViewBox;
+import de.blau.android.util.SavingHelper;
+
+
+public class  BookmarkIO {
+    private BookmarksStorage currentBookmarkStorage = new BookmarksStorage();
+    private ArrayList<BookmarksStorage> bookmarksStorage = new ArrayList<>();
```

Comment:
Use interfaces instead of implementations in declarations, so that should be List not ArrayList.